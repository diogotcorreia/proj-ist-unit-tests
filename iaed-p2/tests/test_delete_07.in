set usr 1
set usr2 2
set usr3 3
delete usr
list
quit