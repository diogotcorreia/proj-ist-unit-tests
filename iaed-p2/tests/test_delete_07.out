usr2
usr3
